pom.xml

```diff
@@ -161,19 +153,31 @@
     <dependency>
       <groupId>org.sonarsource.sonarqube</groupId>
       <artifactId>sonar-ws</artifactId>
-      <version>6.7</version>
+      <version>10.1.0.73491</version>
```

Comment:
Please update the protobuf-java dependency to 3.22.2 (line 78), otherwise there is an enforcer error.